tutorials/basic-topics/sorting/selection-sort.md

```diff
@@ -11,4 +11,110 @@ keywords:
   - selection sort
 ---
 
-<TutorialAuthors names="@TBC"/>
+<TutorialAuthors names="@Bobliuuu"/>
+
+## Overview
+
+Selection sort is a commonly used comparison-based sorting algorithm, that is also very simple to implement. 
```

Comment:
**that is also very simple** to implement. - Instead you can write, **It's simple to implement.**